### ISSUE TITLE
Latest version (1.9 R1) not available in F-Droid

### ISSUE THREAD

**Issue by CryptOwl** [AnySoftKeyboard/AnySoftKeyboard#1276]
The latest release version 1.9 R1 is not available at F-Droid. The latest version available there is 1.8.1333 (released on 2017-06-28): https://f-droid.org/repository/browse/?fdid=com.menny.android.anysoftkeyboard

F-Droid Wiki entry: https://f-droid.org/wiki/page/com.menny.android.anysoftkeyboard
Build log: https://f-droid.org/wiki/page/com.menny.android.anysoftkeyboard/lastbuild_2658

It looks like F-Droid hasn't picked up the latest release for some reason.

**Comment by menny** [comment]
Yes... Sadly, there is an issue with f-droid and pre-release versions of Gradle (the build system we use to build AnySoftKeyboard).
See https://gitlab.com/fdroid/fdroiddata/merge_requests/2812

**Comment by 0rganite** [comment]
The latest version 1.9 R6 is not available in F-Droid.
